query/config_changes.go

```diff
@@ -28,6 +29,13 @@ type CatalogChangesSearchRequest struct {
 	Severity              string    `query:"severity"`
 	IncludeDeletedConfigs bool      `query:"include_deleted_configs"`
 	Depth                 int       `query:"depth"`
+	CreatedByRaw          string    `query:"created_by"`
+	IDs                   string    `query:"ids"`
+	Summary               string    `query:"summary"`
+	Labels                string    `query:"labels"`
```

Comment:
@moshloop Going with `labels` instead of `tags` since `labels` is a superset of it